ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/PeerReputation.java

```diff
@@ -34,19 +34,23 @@
 public class PeerReputation implements Comparable<PeerReputation> {
   static final long USELESS_RESPONSE_WINDOW_IN_MILLIS =
       TimeUnit.MILLISECONDS.convert(1, TimeUnit.MINUTES);
-  static final int DEFAULT_MAX_SCORE = 150;
+  static final int DEFAULT_MAX_SCORE = 200;
+  // how much above the initial score you need to be to not get disconnected for timeouts/useless
+  // responses
+  static final int HAS_BEEN_USEFUL_SCORE_INCREASE = 10;
```

Comment:
nit/suggestion: I feel like it's less work for the reader to use an absolute value rather than an offset.

e.g. 
`static final int HAS_BEEN_USEFUL_THRESHOLD = initialScore + 10`

then...
```
  private boolean peerHasNotBeenUseful() {
    return score < HAS_BEEN_USEFUL_SCORE_THRESHOLD;
  }
```

Might need to do some jiggery pokery depending on how dynamic initial score is to make that work I guess

Reply:
done